File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/SetRobotState.java
package org.firstinspires.ftc.teamcode.commands.low_level;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.opMode.StateMachine;

public class SetRobotState extends InstantCommand {
    public SetRobotState(StateMachine manager, StateMachine._RobotState state){
        super(() -> manager.SetSubsystemState(state));
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/SetRobotState.kt
package org.firstinspires.ftc.teamcode.commands.low_level;

import com.arcrobotics.ftclib.command.InstantCommand;
import org.firstinspires.ftc.teamcode.managers.OuttakeManager

import org.firstinspires.ftc.teamcode.opMode.StateMachine;

class SetRobotState(machine: StateMachine, state: StateMachine._RobotState) : InstantCommand({machine.SetSubsystemState(state)})
